object/virtual_device_list.go

```diff
@@ -919,7 +919,13 @@ func (l VirtualDeviceList) Name(device types.BaseVirtualDevice) string {
 	dtype := l.Type(device)
 	switch dtype {
 	case DeviceTypeEthernet:
-		key = fmt.Sprintf("%d", UnitNumber-7)
+		// Ethernet devices of UnitNumber 7-19 are non-SRIOV. Ethernet devices of 
+		// UnitNumber 45-36 descending are SRIOV
+		if UnitNumber <= 45 && UnitNumber >= 36 {
+			key = fmt.Sprintf("(sriov)-%d", 45-UnitNumber)
```

Comment:
Thanks @sunnycarter 
Can we just use `sriov` without the parens? Just want to leave out any bash special chars for use with govc. Otherwise lgtm

Reply:
Thanks, @dougm, yes I've done that and applied a fixup commit